pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -9278,13 +9278,13 @@ var _ = Describe("v1beta1 inference service controller", func() {
 					},
 				},
 				URL: &apis.URL{
-					Scheme: "http",
+					Scheme: "https",
 					Host:   "raw-auth-default.example.com",
 				},
 				Address: &duckv1.Addressable{
 					URL: &apis.URL{
-						Scheme: "http",
-						Host:   fmt.Sprintf("%s-predictor.%s.svc.cluster.local", serviceKey.Name, serviceKey.Namespace),
+						Scheme: "https",
```

Comment:
Same here

Reply:
This file has too many such instances to feasibly do this. I'll add some info to the source sync documentation for guidelines about merge conflicts.